modules/ROOT/pages/apikit-4-soap-project-task.adoc

```diff
@@ -2,42 +2,38 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:page-aliases: apikit-4-update-wsdl-task.adoc
 
-This process walks you through the creation of a new Mule project based on a SOAP API specification.
+Create a new Mule project based on a SOAP API specification.
 
-Before proceeding, ensure that you meet the xref:apikit-4-soap-prerequisites-task.adoc[Prerequisites for Using APIkit for SOAP].
+Before proceeding, ensure that you meet the xref:apikit-4-soap-prerequisites-task.adoc[prerequisites for using APIkit for SOAP].
 
-. Under the File menu, select *New* > *Mule Project*.
+. In Anypoint Studio, from *File*, select *New* > *Mule Project*.
 +
-image::create-new-mule-project-soap.png["New Mule project window highlighting the project-name field, the runtime engine, the from design center or local files tab, and the expand icon for browsing files."]
+image::create-new-mule-project-soap.png["New Mule project window highlighting the project-name field, the runtime engine, the from design center or local files tab, and the expand icon for browsing files"]
```

Comment:
```suggestion
image::create-new-mule-project-soap.png["New Mule project window that highlights the project-name field, runtime engine, from design center or local files tab, and expand icon for browsing files"]
```

Reply:
Are the "from design center" and "local files" tabs tab names? if so put them in bold. Did you mean to say "from design center"?

Reply:
The capitalization of the tab names here doesn't match the next occurrence.

Reply:
OK! Thanks!